### ISSUE TITLE
Not compatible with PHPUnit 7

### ISSUE THREAD

**Issue by TangRufus** [lucatume/wp-browser#143]
When using `lucatume/wp-browser` v1.22.7 with `PHPUnit` v7.0.2 on PHP 7.2.2, it gives  fatal errors about incompatible subclassing.

```bash
➜ composer test
> codecept run wpunit
Codeception PHP Testing Framework v2.4.0
Powered by PHPUnit 7.0.2 by Sebastian Bergmann and contributors.
PHP Fatal error:  Declaration of Codeception\TestCase\WPTestCase::assertNotFalse($condition, $message = '') must be compatible with PHPUnit\Framework\Assert::assertNotFalse($condition, string $message = ''): void in /Users/xxxxxxx/Code/wp-password-argon-two/vendor/lucatume/wp-browser/src/Codeception/TestCase/WPTestCase.php on line 889
```

When forcing `"phpunit/phpunit": "^6.5"`, everything works again.

**Comment by lucatume** [comment]
For the time being the best approach seems to limit PHPUnit version, in wp-browser own `composer.json` file, to <7.0.
Long term the solution is to load a PHP and PHPUnit aware version of the `WPTestCase` class.

**Comment by lucatume** [comment]
Fixed in `1.22.7.1`
